addons/mail/models/res_config_settings.py

```diff
@@ -59,6 +59,11 @@ class ResConfigSettings(models.TransientModel):
         config_parameter='discuss.tenor_gif_limit',
         help="Fetch up to the specified number of GIF.",
     )
+    google_translate_api_key = fields.Char(
+        'Message Translation API Key',
+        help="A valid Google API key is required to enable message translation. https://cloud.google.com/translate/docs/setup",
+        config_parameter='discuss.google_translate_api_key',
```

Comment:
this could stay just in `mail.` and not discuss

Reply:
Replaced all occurences of "discuss.google_translate_api_key" by "mail.google_translate_api_key"